File: app/src/main/java/com/lumiyaviewer/lumiya/ui/chat/contacts/ActiveChattersFragment.java
package com.lumiyaviewer.lumiya.ui.chat.contacts;

import android.content.Context;
import androidx.loader.app.LoaderManager;
import android.widget.ListAdapter;
import com.lumiyaviewer.lumiya.slproto.users.manager.UserManager;
import com.lumiyaviewer.lumiya.ui.common.UserListFragment;

public class ActiveChattersFragment extends UserListFragment {
    @Override
    protected ListAdapter createListAdapter(Context context, LoaderManager loaderManager, UserManager userManager) {
        return new ActiveChatsListAdapter(getActivity(), userManager);
    }

    @Override
    protected boolean itemsMayBeDismissed() {
        return true;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/ui/chat/contacts/ActiveChattersFragment.kt
package com.lumiyaviewer.lumiya.ui.chat.contacts

import android.content.Context
import android.widget.ListAdapter
import androidx.loader.app.LoaderManager
import com.lumiyaviewer.lumiya.slproto.users.manager.UserManager
import com.lumiyaviewer.lumiya.ui.common.UserListFragment

open class ActiveChattersFragment : UserListFragment() {
    override fun createListAdapter(context: Context, loaderManager: LoaderManager, userManager: UserManager): ListAdapter =
        ActiveChatsListAdapter(activity, userManager)
    override fun itemsMayBeDismissed(): Boolean = true
}
